### DISCUSSION TITLE
Prevent Assets manager dropzone from uploading svg file extension

### DISCUSSION THREAD

**Discussion by nguyenvoanhkhoa1** [GrapesJS/grapesjs#5350] [Q&A] [answered]
Hi guys, is there any way to prevent assets manager dropzone from uploading svg file, since my upload endpoint does not allow to upload svg file. Thanks.

**Comment by artf** [comment, marked as answer]
If you're using the [`Assets.open`](https://grapesjs.com/docs/api/assets.html#open) method you can pass the `accept` option which follows the [`accept` attribute format](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/accept)
